### ISSUE TITLE
OSX (Mojave) - Once you (x) out of the app, you can't restart it without killing it from the terminal.

### ISSUE THREAD

**Issue by Penumbra69** [mooltipass/moolticute#479]
Steps to reproduce:

- Launch Moolticute app, then close it with the (x) on the window.
![image](https://user-images.githubusercontent.com/17106044/66079676-613f6b00-e519-11e9-8fc0-842ca1b1da8e.png)
- Attempt to re-open it via the application (blink, nothing happens)
- Attempt to open it from the daemon on the toolbar (all options are greyed out)
![image](https://user-images.githubusercontent.com/17106044/66079575-2b9a8200-e519-11e9-8e52-ef45bafb1beb.png)

Expected Behavior:
- Launch Moolticute app, then close it with the (x) on the window.
- Re-open it via the toolbar daemon, or launching the app again

The only way I can re-gain the ability to open the application is to go to a terminal, find the daemon and the application, and kill them -9.  

![image](https://user-images.githubusercontent.com/17106044/66079727-7ddba300-e519-11e9-8591-699ece1f5933.png)

After this, I can re-launch the app.

**Comment by Penumbra69** [comment]
Also - if you DO close the app with COMMAND-Q, it crashes but can be reopened:
![image](https://user-images.githubusercontent.com/17106044/66080051-25f16c00-e51a-11e9-82c2-663a8ea01960.png)

**Comment by Penumbra69** [comment]
Further info -- the toolbar daemon is always greyed out, no matter what.. 
No clicks are possible.

**Comment by limpkin** [comment]
tagging @deXol for task after favorites

**Comment by deXol** [comment]
Checked the issue and it looks like a Qt issue.
From Qt 5.12.0 when there is no active (top level) window for MC the systray menu is disabled. However checked from code the systray's context menu is still enabled and the items are also in enabled state, but they are grayed out. I think it will be an OS window handling issue with Qt.
Tested with Qt 5.11.1 and bellow and the issue is not presented, systray menu is working fine.
The only workaround I found is to make MC the active window, when the user clicked on the tray icon: https://github.com/deXol/moolticute/commit/805a91f6a71848887dffa1ce58df96081634edd0 Still not the best user experience, but at least the MC is usable this way.
If I create a minimal project with only systray menu to confirm it, we could open an issue for Qt.
